### ISSUE TITLE
Multiple monitor overview is broken on multiple workspaces

### ISSUE THREAD

**Issue by Ancairon** [paperwm/PaperWM#728]
**Describe the bug**

When I use two monitors in paperwm the overview is broken, because paperwm assigns the workspaces (it seems) in a serial way.

It seems that the top workspace is workspace 1 while the bottom workspace is the second in that array.

![image](https://github.com/paperwm/PaperWM/assets/70198089/de0bafce-fae2-413a-a704-e0b14e921f44)

![image](https://github.com/paperwm/PaperWM/assets/70198089/e865e5c1-4f75-4704-bb13-d6ef024c59d8)

The workspace that has its windows broken is the one out of focus.

**To Reproduce**
1. Have two monitors with a workspace on each
2. Move a window to each workspace
3. Invoke the overview
4. See the bug, one of the two windows will disappear and it will be in the _next_ workspace in that monitor 

**Expected behavior**

Expected is the way gnome does it with workspaces on all monitors, with the same setup I get:

![image](https://github.com/paperwm/PaperWM/assets/70198089/c81100f0-507d-4b66-bc85-92191159b998)

**System information:**

```
Distribution: Ubuntu
GNOME Shell 45.1
Display server: Xorg
Enabled extensions:
- burn-my-windows@schneegans.github.com
- rounded-window-corners@yilozt
- user-theme@gnome-shell-extensions.gcampax.github.com
- panel-corners@aunetx
- nightthemeswitcher@romainvigier.fr
- blur-my-shell@aunetx
- autohide-battery@sitnik.ru
- transparent-top-bar@ftpix.com
- paperwm@paperwm.github.com
- unite@hardpixel.eu
- mediacontrols@cliffniff.github.com
- gnome-fuzzy-app-search@gnome-shell-extensions.Czarlie.gitlab.com
- just-perfection-desktop@just-perfection
- ubuntu-appindicators@ubuntu.com
- ubuntu-dock@ubuntu.com
- window-title-is-back@fthx
```

I use the version from the gnome extension store.

**Comment by shaplyd** [comment]
The issues seems to stem from gnome not supporting isolated workspaces.
https://gitlab.gnome.org/GNOME/mutter/-/issues/37#note_2021664

Could there be a workaround by using only 1 gnome workspace and simulate workspaces by minimising windows like komorebi does on Windows?

**Comment by Thesola10** [comment]
Yeah, that's not really a bug, and I think minimising windows willy-nilly is going to cause way more issues than it fixes. This is inherent to the way PaperWM works and there is no plan to change this for now.

For a more comfortable experience, set Overview to "Scratch Only", as scratch windows are pinned to all workspaces they will properly show up on Overview, although tiled windows will be hidden.
